popSimManu.tex

```diff
@@ -149,18 +153,18 @@ \section*{Introduction}
 This resource, which we call \texttt{stdpopsim}, makes running
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community maintained catalog.
-The \stdpopsim catalog currently contains four organisms: humans,
+The \stdpopsim catalog currently contains five organisms: humans, orangutans
```

Comment:
pongo is still out. will edit on your branch before the merge

Reply:
Okay but it is mentioned elsewhere so be sure to take that out.

Reply:
let's get some more clarity on this. i'll raise an issue.